Packet++/src/TcpReassembly.cpp

```diff
@@ -164,25 +164,18 @@ void TcpReassembly::reassemblePacket(Packet& tcpData)
 	}
 
 	// calculate packet's source and dest IP address
-	IPAddress* srcIP = NULL;
-	IPAddress* dstIP = NULL;
-	IPv4Address srcIP4Addr = IPv4Address::Zero;
-	IPv6Address srcIP6Addr = IPv6Address::Zero;
-	IPv4Address dstIP4Addr = IPv4Address::Zero;
-	IPv6Address dstIP6Addr = IPv6Address::Zero;
+	IPAddress srcAddr, dstAddr;
+	IPAddress* srcIP = &srcAddr;
+	IPAddress* dstIP = &dstAddr;

```

Comment:
I'm not sure why you still need these 2 pointers here

Reply:
It's a temporary code to make less changes. In the next PR all of these will changed.

Reply:
ok sure